components/director/internal/domain/apptemplate/service_test.go

```diff
@@ -419,18 +419,18 @@ func TestService_List(t *testing.T) {
 func TestService_Update(t *testing.T) {
 	// GIVEN
 	ctx := tenant.SaveToContext(context.TODO(), testTenant, testExternalTenant)
-	modelAppTemplate := fixModelApplicationTemplate(testID, testName, []*model.Webhook{})
+	modelAppTemplate := fixModelApplicationTemplate(testID, testName, nil)
```

Comment:
any particular reason for making this nil ?

Reply:
Update in service layer accepts ApplicaitonTemplateUpdateInput which has no webhooks in it. Consequently, the app template passed to the repo layer will contain no Webhooks. Webhooks are updated using the add/update/deleteWebhook mutations.